startup_script/run.sh

```diff
@@ -38,13 +38,26 @@ DATAPROC_VERSION=$(/usr/share/google/get_metadata_value attributes/dataproc-vers
 
 ready=""
 
+function version_ge() ( set +x ;  [ "$1" = "$(echo -e "$1\n$2" | sort -V | tail -n1)" ] ; )
```

Comment:
Similar comment here about whether or not all of the functions are required.

Reply:
probably not called right now, so if you're asking that I not pre-define these functions, I can do that.  But beware that without this function, users will likely use `! version_ge` to mean `version_lt` or `version_le` interchangeably.  There is currently an example of this in bdutil that is breaking the build.

Reply:
Are these functions intended for any kind of reuse by users outside this context? If not, then I think it's fine to remove them.

Reply:
They mirror the same functions in [1].  My concern is that lacking all variants, developers who update this file in the future might make a convoluted and ultimately inaccurate call to this function by negating the results.

It's not a huge concern, though, and we can unbreak the build if it happens again.

I will remove the unused functions in this code.

[1] https://github.com/GoogleCloudDataproc/initialization-actions/blob/989b445b20a2be99b22f169ab9e85f8def9be534/templates/common/util_functions